<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jsonStudio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="ui_grid_layout">
     <item row="0" column="2" rowspan="2">
      <widget class="QPlainTextEdit" name="ui_view_edit">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ui_out_btn">
          <property name="font">
           <font>
            <family>Bahnschrift</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Copy &gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ui_update_btn">
          <property name="font">
           <font>
            <family>Bahnschrift</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt; Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="3">
      <layout class="QHBoxLayout" name="ui_schema_layout">
       <item>
        <widget class="QPushButton" name="ui_load_json_btn">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load JSONâ€¦</string>
         </property>
        </widget>
       </item>
       <item>
       <widget class="QPushButton" name="ui_validate_btn">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Validate</string>
         </property>
       </widget>
      </item>
       <item>
        <widget class="QPushButton" name="ui_clear_btn">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ui_schema_status_label">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>No schema loaded</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
